doc/sphinx/source/install.rst

```diff
@@ -283,6 +283,9 @@ The string after "HSTR:" is the device build hash.
   This is required because pocl binaries contain a device hash, and the LLVM-less
   pocl needs to know which binaries it can load.
 
+**NOTE**: If you've enabled the `almaif <http://portablecl.org/docs/html/almaif.html#usage>`_ 
```

Comment:
Can you fix the URL to a sphinx reference, not an absolute link?

Reply:
sure

Reply:
Do you think adding an example would be nice? like [the original](https://github.com/pocl/pocl/commit/74b46f46be9c8205e1931983164c7f6770f96c2a)?

Reply:
I think it's clear enough now, thanks.

Reply:
great!